neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/embedded/impl/EmbeddedNeo4jSchemaDefiner.java

```diff
@@ -72,6 +97,33 @@ protected void createUniqueConstraintsIfMissing(DatastoreProvider provider, List
 		}
 	}
 
+	private void createCompositeIndex(GraphDatabaseService neo4jDb, CompositeIndexDetails index) {
```

Comment:
Unless if I'm mistaken (and I could be :)), I don't think we should call these methods `Composite` as if it targets only one index, it's a regular index, not a composite one.

So let's rename them to `createIndex()`/`isIndexMissing()`.